src/adagrams.js

```diff
@@ -1,15 +1,121 @@
 export const drawLetters = () => {
   // Implement this method for wave 1
+  const letters = [
+    'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A',
+    'B', 'B',
+    'C', 'C',
+    'D', 'D', 'D', 'D',
+    'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E',
+    'F', 'F',
+    'G', 'G', 'G',
+    'H', 'H',
+    'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I',
+    'J',
+    'K',
+    'L', 'L', 'L', 'L',
+    'M', 'M',
+    'N', 'N', 'N', 'N', 'N', 'N',
+    'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O',
+    'P', 'P',
+    'Q',
+    'R', 'R', 'R', 'R', 'R', 'R',
+    'S', 'S', 'S', 'S',
+    'T', 'T', 'T', 'T', 'T', 'T',
+    'U', 'U', 'U', 'U',
+    'V', 'V',
+    'W', 'W',
+    'X',
+    'Y', 'Y',
+    'Z'
+];
```

Comment:
While it was not a requirement for this project to use a dictionary with letters for keys and number of letters for values to create a list of 98 letters, I would prefer to see you do that in order to get practice iterating over a dictionary to create a new data structure.